Dockerfile

```diff
@@ -73,7 +73,7 @@ ARG TARGETPLATFORM
 # On arm64 we can only install wheel packages.
 RUN case ${TARGETPLATFORM} in \
          "linux/arm64")  pip install --extra-index-url https://download.pytorch.org/whl/cpu/ torch torchvision torchaudio ;; \
-         *)              /opt/conda/bin/conda install -c "${INSTALL_CHANNEL}" -c "${CUDA_CHANNEL}" -y "python=${PYTHON_VERSION}" pytorch torchvision torchaudio "pytorch-cuda=$(echo $CUDA_VERSION | cut -d'.' -f 1-2)"  ;; \
+         *)              pip install --extra-index-url https://download.pytorch.org/whl/${CUDA_VERSION#.}/ torch torchvision torchaudio ;;
```

Comment:
We need INSTALL_CHANNEL logic here to switch between:

https://download.pytorch.org/whl/nightly when INSTALL_CHANNEL==pytorch-nightly
https://download.pytorch.org/whl/test when INSTALL_CHANNEL==pytorch-test
https://download.pytorch.org/whl/  when INSTALL_CHANNEL==pytorch

I believe this is parameter passed from the yml workflow here: https://github.com/pytorch/pytorch/blob/main/.github/workflows/docker-release.yml#L122

Reply:
Do you just want to add another switch to rename those to `whl/nightly/`, `whl/test/` and `whl/` prefixes?

Reply:
Yes I think we can change the variable INSTALL_CHANNEL in https://github.com/pytorch/pytorch/blob/main/.github/workflows/docker-release.yml#L122 to reflect pypi. 
And hence here in Docker address will be: ``https://download.pytorch.org/${INSTALL_CHANNEL}/${CUDA_VERSION#.}/ ``